src/instruments/src/SD/Pages/Door/Door.tsx

```diff
@@ -18,36 +18,16 @@
 
 import './Door.scss';
 import ReactDOM from 'react-dom';
-import { useState } from 'react';
-import { getSimVar, renderTarget, useUpdate } from '../../../util.mjs';
+import React from 'react';
+import { renderTarget } from '../../../Common/defaults';
+import { SimVarProvider, useSimVar } from '../../../Common/simVars';
 
 export const DoorPage = () => {
-    const [cabinDoors, setCabinDoors] = useState({
-        cabin: 0,
-        cabinWarn: false,
-        catering: 0,
-        cateringWarn: false,
-        cargo: 0,
-        cargoWarn: false,
-    });
-
-    const [slides, setSlides] = useState(false);
-    const [oxygen, setOxygen] = useState(false);
-
-    useUpdate(() => {
-        setCabinDoors({
-            cabin: getSimVar('INTERACTIVE POINT OPEN:0', 'percent'),
-            cabinWarn: cabinDoors.cabin > 20,
-            catering: getSimVar('INTERACTIVE POINT OPEN:3', 'percent'),
-            cateringWarn: cabinDoors.catering > 20,
-            cargo: getSimVar('INTERACTIVE POINT OPEN:5', 'percent'),
-            cargoWarn: cabinDoors.cargo > 20,
-        });
-        setSlides(() => (cabinDoors.cabin < 5 && cabinDoors.catering < 5 && cabinDoors.cargoWarn < 5 && getSimVar('LIGHT BEACON ON', 'bool'))
-            || !getSimVar('SIM ON GROUND', 'bool')
-            || getSimVar('ON ANY RUNWAY', 'bool'));
-        setOxygen(() => !getSimVar('L:PUSH_OVHD_OXYGEN_CREW', 'bool'));
-    });
+    const [cabin] = useSimVar('INTERACTIVE POINT OPEN:0', 'percent', 1000);
+    const [catering] = useSimVar('INTERACTIVE POINT OPEN:3', 'percent', 1000);
+    const [cargo] = useSimVar('INTERACTIVE POINT OPEN:5', 'percent', 1000);
+    const [oxygen] = useSimVar('L:PUSH_OVHD_OXYGEN_CREW', 'bool');
```

Comment:
Could also add `1000` for now.

Reply:
Yes, meant to put it back in. Was troubleshooting the SimVars not updating until I remembered the deltaGetTime function in the template.